bql/planner/planner_test.go

```diff
@@ -491,6 +479,94 @@ func TestPlannerQuery(t *testing.T) {
 	}
 }
 
+func TestPlannerConstruct(t *testing.T) {
```

Comment:
Could you add one more test to check that the triples returned are the ones expected, not only the right number?